_data/conferences.yml

```diff
@@ -38,6 +38,29 @@
   sub: ML
   note: '<b>NOTE</b>: Mandatory abstract deadline on Jan 20, 2022. More info <a href=''https://icml.cc/Conferences/2022/''>here</a>.'
 
+- title: IROS
+  hindex: 73
+  year: 2022
+  id: iros22
+  link: https://iros2022.org/
+  deadline: '2022-03-01 23:59:59'
+  timezone: America/Los_Angeles
+  date: October 23-27, 2022
+  place: Kyoto, Japan
+  sub: RO
+
+- title: RSS
+  hindex: 50
+  year: 2022
+  id: rss22
+  link: https://roboticsconference.org/
+  deadline: '2022-01-28 23:59:59'
+  timezone: UTC-12
+  date: June 27-July 1, 2022
+  place: New York City, USA
+  sub: RO
+  paperslink: https://roboticsconference.org/program/papers/
```

Comment:
thanks for suggesting. I think this link is broken. this might be better: https://roboticsconference.org/

Reply:
@omarsar Thanks for the catch! Since that link is already in there, I'll just remove this paperslink entry.

Reply:
Done.